CONTRIBUTING.md

```diff
@@ -14,8 +14,17 @@ git clone git@github.com:Fizzadar/pyinfra.git
 
 # Install the package in editable mode with development requirements
 pip install -e .[dev]
+
+# Configure git to use pre-commit hook
+flake8 --install-hook git
```

Comment:
I think this needs to be `pre-commit install`.

Reply:
Those two commands produce different hooks. I'm ok with either one. The flake8 one was much simpler.

Reply:
Sounds godo to me - we'll need to delete the `.pre-commit-config.yaml` file and `pre-commit` dependency in that case as it won't be required anymore.

Reply:
Do you have a preference? I'm just experimenting.

Reply:
I think `flake8` for now.